packages/vx-xychart/src/classes/DataRegistry.ts

```diff
@@ -0,0 +1,55 @@
+import { AxisScaleOutput } from '@vx/axis';
+import { ScaleConfig } from '@vx/scale';
+import { DataRegistryEntry } from '../types/data';
+
+/** A class for holding data entries */
+export default class DataRegistry<
+  XScaleConfig extends ScaleConfig<AxisScaleOutput>,
```

Comment:
Can change to

```ts
  XScale extends D3Scale<AxisScaleOutput>,
  YScale extends D3Scale<AxisScaleOutput>,
```

because the config types are not used directly